test/system/TODO.md

```diff
@@ -0,0 +1,112 @@
+![PODMAN logo](../../logo/podman-logo-source.svg)
+
+# Overview
+
+System tests exercise podman in the context of a complete, composed environment from
+distribution packages.  It should match as closely as possible to how an end-user
+would experience a fresh-install.  Dependencies on external configuration and resources
+must be kept minimal, and the tests must be generic and vendor-neutral.
+
+The system-tests must execute cleanly on all tested platforms.  They may optionally
+be executed during continuous-integration testing of code-changes, after all other
+testing completes successfully.  For a list of tested platforms, please see [the
+CI configuration file.](../../.cirrus.yml)
+
+
+# Execution
+
+When working from a clone of [the libpod repository](https://github.com/containers/libpod),
+the main entry-point for humans and automation, is `make localsystem`.  When operating
```

Comment:
I'd remove the comma after automation

Reply:
and actually the other in this sentence, be brave, go comma free!

Reply:
lol, but I love my comma's.  They're sooooo soft!